sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -985,8 +985,13 @@ def getRegion(self, format=(TILE_FORMAT_IMAGE, ), **kwargs):
             '-ts', str(int(math.floor(outWidth))), str(int(math.floor(outHeight))),
         ]
 
-        fd, outputPath = tempfile.mkstemp('.tiff', 'tiledGeoRegion_')
-        os.close(fd)
+        outputPath = kwargs.get('output', {}).get('path')
```

Comment:
Do you think it is worth pulling this out to a function and calling that from the three distinct spots?  If not, merge away.

Reply:
Given that this snippet is slightly different in each of the three spots, it probably isn't worth the effort to generalize it as its own function

Reply:
Then merge away.